template1/style.css

```diff
@@ -6,6 +6,11 @@ h1 {
   color: navy;
   margin-left: 20px;
 }
- div{
-   border: 1px solid black;
- }
+.container {
```

Comment:
You replaced the "div" selector with the "container" class selector. There doesn't seem to be any class in the index.html named "container". You will have to change the name of this selector to the class of element you are trying to apply these declarations to.

Here's the vocab I used in this comment: https://www.w3schools.com/css/css_syntax.asp